biohansel/utils.py

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8 -*-
 
 import logging
-from typing import List, Any, Optional, Tuple, Union
+from typing import List, Any, Tuple, Union
 
 import os
```

Comment:
"os" is a core module, perhaps it should come right after logging to maintain the proper import format

Reply:
They're all built-ins in that file actually, but they could definitely be organized a bit better. Changed in f1df3adf22308cc9bd76223c80c53d000c42a71d